bfasst/utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Import general functions into the utils namespace"""
+from .general import *
```

Comment:
I'm not sure about this approach.  Isn't it better to just import utility functions as needed?

Otherwise you have code that does this:
```python
import bfasst.utils

<use function where it's not clear where it originated>
```